contents/blog/save-engineers-time.md

```diff
@@ -0,0 +1,77 @@
+
+As an example, the crypto wallet [Phantom](https://posthog.com/customers/phantom) uses PostHog Feature Flags as kill switches for all their features. This is because they can't deploy a new version of their Chrome Extension on demand. Feature Flags give them control of the user experience, limit feature failure, and save time.
+
+On top of Feature Flags, PostHog includes several other testing tools that are simple for you to set up and easy for non-engineers to use. They include [Session Recording](https://posthog.com/product/session-recording), [Heatmaps](https://posthog.com/product/heatmaps), [Multivariate Testing](https://posthog.com/product/experimentation-suite), and more. These create a positive feedback loop in product decisions that help features get tested faster. The team gets data on the quality of their decision and this helps them improve and speed up the next one.
+
+## Simple, customizable data ingestion
+
+When implementing Product Analytics, many services require significant work to implement and customize. They often require writing detailed requests in your app and ensuring they are handled properly in your analytics platform. PostHog makes it simple to capture events and gain data on usage. At its [simplest](https://posthog.com/docs/integrate), it requires either a one-line [snippet](https://posthog.com/docs/integrate#snippet) or an API request with the wanted data object.
+
+You don't need to spend time implementing a library for sending those API requests because we've already built them. We have [libraries](https://posthog.com/docs/integrate#libraries) in client and server languages like Javascript, Python, and Ruby. As well, we have several integrated ingestion [apps](https://posthog.com/docs/integrate#apps) like Amazon Kinesis, Shopify, and Zapier. This doesn't prevent you from implementing your own library with our API but saves you time if you need it.
+
+We automatically capture data from wherever it is sent. No need to do elaborate configurations of each instance. You won't have to read through pages of documentation to understand how to capture an event correctly. Within PostHog, the data can be sliced, filtered, modified, and analyzed. The people who care more about the data (like the ones making product decisions) can analyze it how they like. This saves you time on implementation and customization.
+
+PostHog helps companies capture more data faster. For example, [Saga](https://posthog.com/customers/saga)found that PostHog saw 40-50% more events than Mixpanel because of self-hosting. We also managed to track 20-30% more users. By capturing more data, decisions are made faster and more accurately. This helped Saga find a missing step in their tutorials, leading to increased retention of 70-80%. 
```

Comment:
```suggestion
PostHog helps companies capture more data faster. For example, [Saga](https://posthog.com/customers/saga) found that PostHog captured 40-50% more events than Mixpanel because a self-hosted deployment of PostHog avoids reliance on frequently blocked third-party cookies. We also managed to track 20-30% more users. By capturing more data, decisions are made faster and more accurately. This helped Saga find a missing step in their tutorials, leading to increased retention of 70-80%. 
```

Important to be specific on this point, so it can't be read with the opposite meaning.